Rebaseline intl-displaynames-v2

danlliu opened the pull request:
#### 863014818888034ab85830dc4ad115b0ce8b0556
<pre>
Rebaseline intl-displaynames-v2
<a href="https://bugs.webkit.org/show_bug.cgi?id=296204">https://bugs.webkit.org/show_bug.cgi?id=296204</a>
<a href="https://rdar.apple.com/154874458">rdar://154874458</a>

Reviewed by Yusuke Suzuki.

Update test expectations because of ICU changes.

* JSTests/stress/intl-displaynames-v2.js:
(displayNames.of):
(shouldBe):

Canonical link: <a href="https://commits.webkit.org/297622@main">https://commits.webkit.org/297622@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/355b4b5f833f36a610528c5f8b6c2a7c8358552f

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/112325 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/32056 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/22534 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/118404 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/62689 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/32709 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/40620 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/85327 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/36019 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/115272 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/26099 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/101059 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/65758 "Passed tests") | 
| | [⏳ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-WPT-WK2-Tests-EWS "Waiting to run tests") | | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/62249 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/104839 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/95483 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/19274 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/121729 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/110939 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/39399 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/29322 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/94135 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/39780 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/97301 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/93961 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24027 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/39205 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/17001 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/35449 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/39287 "Built successfully") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/135170 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/38922 "Built successfully") | | [❌ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/36325 "Found 2 new JSC stress test failures: wasm.yaml/wasm/stress/jitless-wasm-to-js-should-follow-to-wasm-call-frame-convention-2.js.default, wasm.yaml/wasm/stress/jitless-wasm-to-js-should-follow-to-wasm-call-frame-convention.js.default (failure)") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/42259 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/40665 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

Constellation commented on JSTests/stress/intl-displaynames-v2.js:
```diff
@@ -47,7 +47,7 @@ function shouldThrow(func, errorMessage) {
     shouldBe(dn.of("ethioaa"), "Ethiopic Amete Alem Calendar");
     shouldBe(dn.of("japanese"), "Japanese Calendar");
-    shouldBe(dn.of("dangi"), "Dangi Calendar");
+    shouldBe(dn.of("dangi"), "Korean Calendar");
```
For newer data, please accept both since we need to support some builds with older ICU.

Constellation commented on JSTests/stress/intl-displaynames-v2.js:
```diff
@@ -57,7 +57,7 @@ function shouldThrow(func, errorMessage) {
     shouldBe(dn.of("ethioaa"), "埃塞俄比亚阿米特阿莱姆日历");
     shouldBe(dn.of("japanese"), "和历");
-    shouldBe(dn.of("dangi"), "檀纪历");
+    shouldBe(dn.of("dangi"), "韩国历");
```
Ditto.

danlliu committed:
Rebaseline intl-displaynames-v2
https://bugs.webkit.org/show_bug.cgi?id=296204
rdar://154874458

Reviewed by Yusuke Suzuki.

Update test expectations because of ICU changes.

* JSTests/stress/intl-displaynames-v2.js:
(displayNames.of):
(shouldBe):

Canonical link: https://commits.webkit.org/297622@main
